crates/bifrost/src/providers/replicated_loglet/tasks/digests.rs

```diff
@@ -247,8 +267,10 @@ impl Digests {
                     %offset,
                     "Record has been repaired"
                 );
+                self.num_fixups += 1;
                 // record has been fully replicated.
-                self.update_start_offset(offset);
+                self.update_start_offset(offset.next());
```

Comment:
This is one the actual bug that was fixed

Reply:
Imagine there's no off-by-one errors
It's easy if you try
No hell below us
Above us, only sky